### ISSUE TITLE
Open files with external apps by default

### ISSUE THREAD

**Issue by Android-PowerUser** [hardcore-sushi/DroidFS#83]
To open files with other apps you have to mark them first, then menu, then external app and then possibly chose the app. It would be better to be able to set certain file types to always be opened with a certain app.

**Comment by hardcore-sushi** [comment]
Yes, good idea. We could also suggest the user to remember file extension association even for internal opening. But this would require a section in the settings to edit this preferences in case the user want to remove the association.
